### PR TITLE
Add byte threshold and remove approximate threshold for load base split

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Add byte threshold and remove approximate threshold for load base split

Add byte threshold, because it will cover more case, such as there is a region with 1k qps and 30MB/s byte rate

Remove approximate threshold, because it is inaccurate

### Check List 

Tests 

- Unit test
- Manual test
```
set config tikv split.qps-threshold=50000;
drop database load_base_split;
create database load_base_split;
```
```
sysbench --rand-type=uniform --table_size=100000 --range_size=5000 --point_selects=0 --sum_ranges=0 --order_ranges=0 --distinct_ranges=0 --config-file=config oltp_read_only prepare
sysbench --rand-type=uniform --table_size=100000 --range_size=5000 --point_selects=0 --sum_ranges=0 --order_ranges=0 --distinct_ranges=0 --config-file=config oltp_read_only run
```

image
image

### Release note 
- No Release note

### CODE REVIEW

**Review by nolouch — APPROVED** [review]
lgtm

**Review by HunDunDM — APPROVED** [review]

File: components/raftstore/src/store/worker/split_config.rs [hunk 1]
```diff
@@ -25,9 +24,8 @@ pub struct SplitConfig {
     pub split_contained_score: f64,
     pub detect_times: u64,
     pub sample_num: usize,
-    pub sample_threshold: i32,
-    pub size_threshold: u64,
-    pub key_threshold: u64,
+    pub sample_threshold: u64,
+    pub byte_threshold: usize,
```
Comment (cosven) [inline components/raftstore/src/store/worker/split_config.rs:28]: I think this change breaks backward compatibility.

If a user set the config `size_threshold` in their config files and it does not take effects after they upgrade to the newer version.

Comment (cosven) [reply components/raftstore/src/store/worker/split_config.rs:28]: I created an issue https://github.com/tikv/tikv/issues/9816

**Pull request merged**